cmd/experimental/kjobctl/pkg/cmd/describe/helpers.go

```diff
@@ -150,6 +150,37 @@ func printLabelsMultilineWithIndent(w describehelper.PrefixWriter, initialIndent
 		w.Write(IndentLevelZero, "%s=%s\n", key, labels[key])
 	}
 }
+func printAnnotationsMultiline(w describehelper.PrefixWriter, title string, annotations map[string]string) {
```

Comment:
I'm wondering. Can we just rename `printLabelsMultiline`? It looks like code the same? Or I missed something?